app/javascript/shared/mixins/specs/whatsappTemplates/whatsappTemplates.spec.js

```diff
@@ -18,45 +19,45 @@ const config = {
 };
 
 describe('#WhatsAppTemplates', () => {
-  it('returns all variables from a template string', () => {
+  it('returns all variables from a template string', async () => {
     const wrapper = shallowMount(TemplateParser, {
       ...config,
       propsData: { template: templates[0] },
     });
+    await nextTick();
     expect(wrapper.vm.variables).toEqual(['{{1}}', '{{2}}', '{{3}}']);
   });
 
-  it('returns no variables from a template string if it does not contain variables', () => {
+  it('returns no variables from a template string if it does not contain variables', async () => {
     const wrapper = shallowMount(TemplateParser, {
       ...config,
       propsData: { template: templates[12] },
     });
+    await nextTick();
     expect(wrapper.vm.variables).toBeNull();
   });
 
-  it('returns the body of a template', () => {
+  it.skip('returns the body of a template', async () => {
```

Comment:
@fayazara had to skip this test

The issue is highlighted here: https://github.com/vuejs/test-utils/issues/2196
Solution was made but only works for vue 3: https://github.com/vuejs/test-utils/issues/2196

CC: @pranavrajs

Reply:
Fixed, just had to return it in the `setup`